from0k2bp.tex

```diff
@@ -552,11 +562,11 @@
 the aforementioned single bit.
 \end{quote}
 
-For more details on this basic (if rather bizarre at first) reasoning
-for the construction of zero knowledge proofs, refer to e.g. ,
+For more details on this basic reasoning (rather bizarre at first) 
```

Comment:
Again, this stylistic change I don't really agree with, not that it hugely matters.

Reply:
Reverted.